make.js

```diff
@@ -339,6 +339,10 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
 function getResultActions(tabbing, apiCall, isInstanceApi) {
     if (apiCall.url === "/Authentication/GetEntityToken")
         return tabbing + "context->HandlePlayFabLogin(\"\", \"\", outResult.Entity->Id, outResult.Entity->Type, outResult.EntityToken);\n";
+    if (apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
+        return tabbing + "context->HandlePlayFabGameServerLogin(\"\", \"\", outResult.EntityToken->Entity->Id, outResult.EntityToken->Entity->Type, outResult.EntityToken->EntityToken);\n";
+    if (apiCall.url === "/GameServerIdentity/Delete")

```

Comment:
I don't think we should add any special handling for Delete. We risk clearing a valid entity token in some edge cases (like if the cached entity token had been changed back to a title entity token, or another game_server)